rma_receipt_location/models/rma.py

```diff
@@ -0,0 +1,32 @@
+# Copyright NuoBiT - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from odoo import api, fields, models
+
+
+class Rma(models.Model):
+    _inherit = "rma"
+
+    receipt_location_id = fields.Many2one(
+        comodel_name="stock.location",
+        string="Receipt Location",
+        compute="_compute_receipt_location_id",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("partner_id")
+    def _compute_receipt_location_id(self):
+        for rec in self:
+            rec.receipt_location_id = rec.partner_id.property_stock_customer
+
+    def _prepare_picking(self, picking_form):
+        picking_form.origin = self.name
+        picking_form.partner_id = self.partner_shipping_id
+        picking_form.location_id = self.receipt_location_id
+        picking_form.location_dest_id = self.location_id
+        with picking_form.move_ids_without_package.new() as move_form:
+            move_form.product_id = self.product_id
+            move_form.product_uom_qty = self.product_uom_qty
+            move_form.product_uom = self.product_uom
```

Comment:
I have tried to do this, but it didn't work. That's why I overwrite the content. I have created a module in Nuobit because I have doubts about whether this IMP should be pushed to the OCA

```python
def _prepare_picking(self, picking_form):
    default = {
        "name": self.name,
    }
    self_copy = self.new(self.copy_data(default=default)[0])
    self_copy.partner_shipping_id.property_stock_customer = self.receipt_location_id
    return super(Rma, self_copy)._prepare_picking(picking_form)

```

Reply:
Ok, I agrree with you regarding not pushed to OCA.
We can discuss the "overwrite" thing you mentiones